quetz/tests/test_health.py

```diff
@@ -0,0 +1,11 @@
+from fastapi.testclient import TestClient
```

Comment:
Do you know if we can easily add a test case where the health endpoints don't return 200?

Reply:
I think with the FastAPI test client, this will be difficult. We would probably need to set up a proper integration test with a running Quetz server instance.

Reply:
It is possible to [override dependencies for testing](https://fastapi.tiangolo.com/advanced/testing-dependencies/), that's kind of the whole point of Dependency Injection. Now the question is exactly how to write the function. Then, that'd illuminate to properly implement the endpoint and the tests.

Reply:
Setting up an integration test will be complicated, so I wouldn't include that in this PR.